src/legacy/core_plugins/metrics/public/components/vis_types/table/vis.js

```diff
@@ -48,10 +49,9 @@ class TableVis extends Component {
     this.renderRow = this.renderRow.bind(this);
   }
 
-  renderRow(row) {
-    const { model } = this.props;
-    const rowId = row.key;
-    let rowDisplay = rowId;
+  renderRow = (isDateTypeField = false) => row => {
+    const { model, dateFormat } = this.props;
```

Comment:
If you don't find a feasible way to use `DateFormat` as described in the below comment, please, add `dateFormat` to the `propTypes` of this component.